src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const priceADay = 40;
+  const LONG_TERM = 7;
+  const LONG_DISCOUNT = 50;
+  const MIDDLE_TERM = 3;
+  const MIDDLE_DISCOUNT = 20;
+  const TotalPrice = priceADay * days;
```

Comment:
The variable `TotalPrice` should be named using camelCase to follow JavaScript naming conventions. Consider renaming it to `totalPrice`.